nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -126,10 +98,10 @@
         .build();
 
     protected static final PropertyDescriptor INSERT_OPERATION = new Builder()
-        .name("Insert Operation")
+        .name("Kudu Operation Type")
```

Comment:
Changing the name will break existing processors. Instead add `.displayName("Kudu Operation Type")` to change the displayed name without changing the property name.